manifests/plugin/container.pp

```diff
@@ -27,7 +19,7 @@
             'url' => "${pulpcore::apache::api_base_url}${registry_version_path}",
           },
         ],
-        'request_headers' => $api_default_request_headers + $api_additional_request_headers,
+        'request_headers' => $pulpcore::apache::api_default_request_headers + $pulpcore::apache::api_additional_request_headers,
```

Comment:
I debated introducing a new variable that has both these values, but decided against it to keep this smaller.